cypress/integration/common/fixtures.js

```diff
@@ -6,6 +6,7 @@ const {
 // but only if we're not running under CI. In CI we leave it intact for 
 // the next scenario, to simulate the user journey
 After({ tags: "@CleanupFixtures" }, () => {
+    cy.log('cleaning up fixtures');
```

Comment:
Do we still want this in the log?

Reply:
It doesn't hurt. It also makes it easier to see when the fixtures aren't being cleaned; without it, you just get an After > document log entry which doesn't explain where it comes from. I found it useful to see an explicit "I've cleaned up your fixtures" message.